docfx_project/ziti/quickstarts/network/hosted.md

```diff
@@ -1,72 +1,75 @@
-# Host Ziti Anywhere
+# Host OpenZiti Anywhere
 
-You can absolutely choose to host your [Ziti Network](xref:zitiOverview#overview-of-a-ziti-network) anywhere you like.
-It is not necessary for the server to be on the open internet. If it works better for you to deploy Ziti on your 
-own network, great, do that.  The only requirement to be aware of is that every piece of the a network will need to 
-be able to communicate to the controller at least one edge router. If these components are all on your local network,
-that's perfectly fine. Just remember, the only components able to participate on this network will need to be able to
-contact the controller and at least one edge router.
+You can absolutely choose to host your [OpenZiti Network](xref:zitiOverview#overview-of-a-ziti-network) anywhere you like.
+It is not necessary for the server to be on the open internet. If it works better for you to deploy OpenZiti on your
+own network, great, do that.  The only requirement to be aware of is that every piece of the a network will need to
+be able to communicate to the OpenZiti Edge i.e. the control plane that is composed of the controller's edge-client REST API and

```

Comment:
i.e. has an academic tone and is not conversational. I don't want the tone to be academic. it needs to be easy to read, and more casual than formal

Reply:
"OpenZiti Edge i.e. the control plane" - i really prefer the previous words.